### DISCUSSION TITLE
ParquetLoader ERRORS

### DISCUSSION THREAD

**Discussion by aycaramba1224** [visgl/loaders.gl#3040] [Q&A]
![image](https://github.com/visgl/loaders.gl/assets/60164178/f4f1c57e-beac-4c3a-95f9-a5e509b69769)

'https://unpkg.com/@loaders.gl/parquet@4.2.1/dist/parquet-worker.js' is wrong url!
'https://unpkg.com/@loaders.gl/parquet@4.2.1/dist/workers/parquet-worker.js' is correct url!

**Comment by JfrAziz** [comment]
any updates on this?, I encounter the same issue when access :

`importScripts('https://unpkg.com/@loaders.gl/wkt@4.3.0/dist/wkb-worker.js');`

correct url
`importScripts('https://unpkg.com/@loaders.gl/wkt@4.3.0/dist/workers/wkb-worker.js');`
